src/cookbooks/deploying-containers.md

```diff
@@ -0,0 +1,49 @@
+# Building and Deploying Containers to Google Container Registry (GCR) with CircleCI
+
+The following cookbook describes how to set up automated build and deployment for containers with CircleCI, a useful pattern for scheduling custom jobs in Google Kubernetes Engine.
+
+<!-- toc -->
+
+## Assumptions
+
+* Your GitHub project's repository has a working Dockerfile at its root
+    * If your file is not named `Dockerfile` or not located in the repo root, see the docs for the [CircleCI GCP-GCR orb](https://circleci.com/orbs/registry/orb/circleci/gcp-gcr) for additional configuration
+* The repository is in the `mozilla` GitHub org (or another org with a paid CircleCI account)
+
+
+## Steps
+
+
+### On GCP
+
+* Create your GCP sandbox project and enable Container Registry (https://console.cloud.google.com/gcr/images/\<your-project-id\>)
+* Create a service account, give it the “Storage Admin” role and create a key
+	* Console Link: [Google Cloud Platform](https://console.cloud.google.com/iam-admin/serviceaccounts)
+	* Additional documentation: [Configuring access control  |  Container Registry Documentation](https://cloud.google.com/container-registry/docs/access-control?hl=en_US)
+
+
+### On CircleCI
+
+* **IMPORTANT SECURITY STEP**
+    * Go to your project’s CircleCI Advanced Settings Page (e.g. https://circleci.com/gh/mozilla/pensieve/edit#advanced-settings) and TURN OFF “*Pass secrets to builds from forked pull requests*”
```

Comment:
```suggestion
    * Go to your project’s CircleCI Advanced Settings Page (e.g. https://circleci.com/gh/mozilla/pensieve/edit#advanced-settings) and make sure that the "*Pass secrets to builds from forked pull requests*" option is TURNED OFF
```

This should be turned off by default, so the action here should usually just be verifying rather than actually toggling the option.